### DISCUSSION TITLE
page load event

### DISCUSSION THREAD

**Discussion by airman5573** [woowacourse-study/2022-woowahan-bansanghwe#2] [JavaScript]
DOMContentLoaded, load, beforeunload, unload를 각각 설명해 주세요

**Comment by wonsss** [comment]
https://ko.javascript.info/onload-ondomcontentloaded

**Comment by rladpwl0512** [comment]
- `DOMContentLoaded`, `load`의 차이는? 
- `beforeunload`, `unload`의 차이는?
